src/thermal_conductivity/options.f90

```diff
@@ -48,94 +44,89 @@ subroutine parse(opts)
     logical :: dumlog
     real(flyt) :: f0
     real(flyt), dimension(3) :: dumflytv
+    integer :: i
 
     ! basic info
     call cli%init(progname='thermal_conductivity', &
                   authors=lo_author, &
                   version=lo_version, &
                   license=lo_licence, &
                   help='Usage: ', &
-                  description='Calculates the lattice thermal conductivity from the iterative solution of the &
-                              &phonon Boltzmann equation. In addition, cumulative plots and raw data dumps &
-                              &of intermediate values are available.', &
-                  examples=["mpirun thermal_conductivity --temperature 300                              ", &
-                            "mpirun thermal_conductivity -qg 15 15 15 --temperature_range 200 600 50    ", &
-                            "mpirun thermal_conductivity --integrationtype 2 -qg 30 30 30 --max_mfp 1E-6"], &
+                  description='Calculates the lattice thermal conductivity in the&
+                              & mode-coupling formalism, including collective and off-diagonal&
+                              & contributions up to fourth-order interactions.',&
+                  examples=["mpirun thermal_conductivity --temperature 300                        ", &
+                            "mpirun thermal_conductivity --fourthorder  -qg 30 30 30 -qg4ph 4 4 4 "], &
                   epilog=new_line('a')//"...")
     ! real options
     call cli%add(switch='--readiso', &
                  help='Read the isotope distribution from `infile.isotopes`.', &
                  help_markdown='The format is specified [here](../page/files.html#infile.isotopes).', &
                  required=.false., act='store_true', def='.false.', error=lo_status)
     if (lo_status .ne. 0) stop
-    cli_qpoint_grid
     call cli%add(switch='--integrationtype', switch_ab='-it', &
-                 help='Type of integration for the phonon DOS. 1 is Gaussian, 2 adaptive Gaussian and 3 Tetrahedron.', &
-                 required=.false., act='store', def='2', choices='1,2,3', error=lo_status)
+                 help='Type of integration for the phonon DOS. 1 is Gaussian, 2 adaptive Gaussian.', &
+                 required=.false., act='store', def='2', choices='1,2', error=lo_status)
     if (lo_status .ne. 0) stop
+    call cli%add(switch='--nothirdorder', &
+                 help='Do not consider third order contributions to the scattering.', &
+                 required=.false., act='store_true', def='.false.', error=lo_status)
+    if (lo_status .ne. 0) stop
+    call cli%add(switch='--fourthorder', &
+                 help='Consider four-phonon contributions to the scattering.', &
+                 required=.false., act='store_true', def='.false.', error=lo_status)
+    if (lo_status .ne. 0) stop
+    cli_qpoint_grid
     call cli%add(switch='--sigma', &
```

Comment:
Keeping it with the same name is more consistent with what is happening in other programs (like lineshape).
But I will change the documentation so that it is consistent with lineshape